WordPress/src/main/java/org/wordpress/android/ui/prefs/AppPrefs.java

```diff
@@ -125,6 +130,7 @@
         SHOULD_AUTO_ENABLE_GUTENBERG_FOR_THE_NEW_POSTS_PHASE_2,
         GUTENBERG_OPT_IN_DIALOG_SHOWN,
         GUTENBERG_FOCAL_POINT_PICKER_TOOLTIP_SHOWN,
+        GUTENBERG_BLOCK_TYPE_IMPRESSIONS,
```

Comment:
Because this is a `DeleteablePrefKey` this will get reset if the user logs in and logs out. This is a bit of an edge case, but a user who regularly switches accounts would have the impressions reset each time they switched. I wonder if we would be better off making this an `UndeleteablePrefKey`.

Reply:
@mchowning good thought. Persisting the value throughout the life of the app install was my original intent. I agree that resetting this value when a user logs out would likely be unexpected. I will update the value to be an `UndeleteablePrefKey`.

Reply:
Addressed in https://github.com/wordpress-mobile/WordPress-Android/pull/15038/commits/03dee0565ec8325a0f515dd94be5a298f00badd9.